libs/chlorophyll/scss/components/utility/_colors.scss

```diff
@@ -50,15 +51,15 @@ $colors: (
 );
 @each $key, $variant in $colors {
   .text-#{$key} {
-    color: map.get($variant, 'color') !important;
-    --color: #{map.get($variant, 'color')} !important;
+    color: map.get($variant, 'color');
+    --color: #{map.get($variant, 'color')};
   }
 
   .bg-#{$key} {
-    background: map.get($variant, 'color') !important;
-    --background: map.get($variant, 'color') !important;
+    background: map.get($variant, 'color');
+    --background: map.get($variant, 'color');
     @if (map.get($variant, 'contrast') != null) {
-      color: map.get($variant, 'contrast') !important;
```

Comment:
Good riddance to those `!important` :D
This may be a breaking change for some consumers though, so we should mention it in the release notes.

Reply:
I think it's fine if we release it at the same time as #1045, which is also potentially breaking!

The use of `!important` here is probably a vestigial inheritance from Bootstrap, where utility classes usually had this to overcome specificity rules. Not sure if that is even needed here, but if it is, it would probably make more sense to use a cascade layer for that instead (i.e put all utility classes under `@layer utilities`, or something like that)

Reply:
I'll leave that feature/fix for when we know more what chlorophyll will be in the future - just sticking to actual bug fixes for now. I feel that if I go down this rabbit hole I might not come up until a couple of weeks.... 🐰